admin/javascript/LeftAndMain.js

```diff
@@ -150,7 +150,7 @@ jQuery.noConflict();
 			}
 
 			// Show message (but ignore aborted requests)
-			if(xhr.status !== 0 && msg && $.inArray(msg, ignoredMessages)) {
+			if(xhr.status !== 0 && msg && $.inArray(msg, ignoredMessages) != -1) {
```

Comment:
You said in https://travis-ci.org/silverstripe/silverstripe-framework/builds/125725811 that it would be `== -1` not `!= -1`